README.md

```diff
@@ -4,221 +4,501 @@
+#### kv Backend read Parameters
 
-For detailed info on these parameters please visit [Stream](https://www.iguazio.com/docs/concepts/latest-release/streams) docs.
+- <a id="method-read-kv-param-reset_index"></a>**reset_index** &mdash; `bool` &mdash; Reset the index. When set to `false` (default), the DataFrame will have the key column of the v3io kv as the index column.
+  When set to `true`, the index will be reset to a range index.
+- <a id="method-read-kv-param-max_in_message"></a>**max_in_message** &mdash; `int` &mdash; Maximal number of rows per message
+- <a id="method-read-kv-param-"></a>**sharding_keys** &mdash; `[]string` (**Experimental**) &mdash; a list of specific sharding keys to query, for range-scan formatted tables only.
+- <a id="method-read-kv-param-segments"></a>**segments** &mdash; `[]int64` (**Not yet supported**)
+- <a id="method-read-kv-param-total_segments"></a>**total_segments** &mdash; `int64` (**Not yet supported**)
+- <a id="method-read-kv-param-sort_key_range_start"></a>**sort_key_range_start** &mdash; `string` (**Not yet supported**)
+- <a id="method-read-kv-param-sort_key_range_end"></a>**sort_key_range_end** &mdash; `string` (**Not yet supported**)
+
+For detailed information about these parameters, refer to the platform's NoSQL documentation.
 
 Example:
 ```python
-df = client.read(backend='stream', table='mytable', seek='latest', shard_id='5')
+df = client.read(backend="kv", table="mytable", filter="col1>666")
 ```
 
-### Delete
-Deletes a table of a specific backend.
+<a id="method-read-params-stream"></a>
+#### stream Backend read Parameters
+
+- <a id="method-read-stream-param-seek"></a>**seek** &mdash; `string` &mdash; valid values:  `"time" | "seq"/"sequence" | "latest" | "earliest"`.
+  <br/>
+  If the `"seq"|"sequence"` seek type is set, you need to provide the desired record sequence ID via the [`sequence`](#method-read-stream-param-sequence) parameter.
+  <br/>
+  If the `time` seek type is set, you need to provide the desired start time via the `start` parameter.
+- <a id="method-read-stream-param-shard_id"></a>**shard_id** &mdash; `string`
+- <a id="method-read-stream-param-sequence"></a>**sequence** &mdash; `int64` (Optional)
+
+For detailed information about these parameters, refer to the [platform streams documentation](https://www.iguazio.com/docs/concepts/latest-release/streams).
 
 Example:
 ```python
-df = client.delete(backend='<backend>', table='mytable')
+df = client.read(backend="stream", table="mytable", seek="latest", shard_id="5")
 ```
 
-#### backend specific parameters
-##### TSDB
-* start: string - delete since start
-* end: string - delete since start
+<a id="method-delete"></a>
+### delete Method
+
+Deletes a data collection (table/stream) in a platform data container according to the configured backend.
+<br/>
+The `kb` backend also supports an optional [`filter`](#method-delete-kv-param-filter) parameter that can be used to delete only specific items in a NoSQL tables.
```

Comment:
`kb` -> `kv`

Reply:
Fixed; will be committed shortly to the PR branch.